package.json

```diff
@@ -27,17 +27,19 @@
   "dependencies": {
     "axios": "^0.13.1",
     "babel-polyfill": "^6.9.1",
-    "react": "^15.2.1",
+    "npm": "^6.4.1",
```

Comment:
Syriously ? you add npm to local dependeces ? npm global package. Please remove this.